INPTPZ1/Program.cs

```diff
@@ -23,274 +21,153 @@ namespace INPTPZ1
     /// </summary>
     class Program
     {
+        private static int canvasSide, iteration, idOfRoot;
+        private static double xmin, ymin, xmax, ymax, xstep, ystep;
+        private static Bitmap bitmapCanvas;
+        private static Color[] colors;
+        private static Polynome polynome, derivationOfPolynome;
+        private static List<ComplexNumber> roots;
+        private static ComplexNumber chosenComplex;
+
+        private static void Initialize()
+        {
+            bitmapCanvas = new Bitmap(canvasSide, canvasSide);
+            xstep = (xmax - xmin) / canvasSide;
+            ystep = (ymax - ymin) / canvasSide;
+
+            colors = new Color[] { Color.Red, Color.Green, Color.Blue };
+            roots = new List<ComplexNumber>();
+            polynome = new Polynome(new List<ComplexNumber>
+            {
+                new ComplexNumber { RealPart = 1},
+                ComplexNumber.Zero,
+                ComplexNumber.Zero,
+                new ComplexNumber { RealPart = 1},
+            });
+            derivationOfPolynome = polynome.Derivate();
+            Console.WriteLine("f(x) = " + polynome);
+            Console.WriteLine("f'(x) = " + derivationOfPolynome);
+        }
+
         static void Main(string[] args)
```

Comment:
Too long and too much complex method

Reply:
reduced size of main method by introducing new method